packages/cosmos_ui_components/lib/components/cosmos_elevated_button.dart

```diff
@@ -5,18 +5,24 @@ class CosmosElevatedButton extends StatelessWidget {
   final VoidCallback? onTap;
   final String text;
   final Widget? suffixIcon;
+  final bool outlined;
 
   const CosmosElevatedButton({
     Key? key,
     this.onTap,
     this.text = "",
     this.suffixIcon,
+    this.outlined = false,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return ElevatedButton(
       onPressed: onTap,
+      style: ElevatedButton.styleFrom(
+        primary: Theme.of(context).colorScheme.secondary,
+        onPrimary: Theme.of(context).colorScheme.onSecondary,
```

Comment:
@andrzejchm 
Removed the outline thing. Just keeping this color thing because the button was acting weird when switching between dark/light modes.

Reply:
Instead of playing with colors here, I think its better to update the cosmos theme accordingly in `CosmosAppTheme` class. right now we're setting secondary color to primary, which might cause some headscratching when someone decides to alter theme of their apps